flake.nix

```diff
@@ -5,35 +5,37 @@
     nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
   };
 
-  outputs = { self, nixpkgs }:
-    let
-      system = "x86_64-linux";
-      pkgs = import ./pkgs.nix {
-        pkgs = nixpkgs.legacyPackages.${system};
-        inherit system;
-      };
-    in
-    {
-      devShells.${system} = {
-        default = pkgs.mkShell {
-          buildInputs = with pkgs; [
-            holochain_0-4
-            lair-keystore_0-4-5
-            hc_0-4
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
```

Comment:
`eachDefaultSystem` does not include windows and it includes linux i686 which we don't support.

I tried to make a custom list without success.